Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM mhart/alpine-node:6
+MAINTAINER code.vire@gmail.com
+
+RUN npm install -g yarn
+
+COPY package.json /tmp
+RUN cd /tmp &&\
+    yarn &&\
+    mkdir /app &&\
+    mv node_modules /app/node_modules &&\
+    rm package.json
+
+WORKDIR /app
+COPY . /app
+
+ARG REACT_APP_BACKEND_URL # env variable to specify API URL durin build
```

Comment:
had to remove comment from this line to make it work (otherwise I've got an error: "ARG requires exactly one argument definition")